### ISSUE TITLE
How to manually check if one position or row has correct masking?

### ISSUE THREAD

**Issue by Leo-T-Zang** [pytorch-labs/attention-gym#66]
Hi,

I wonder if we can manually verify attention mask patterns during testing. While I can visualize masks by printing them as strings, I'm looking to add proper test assertions.

- How to assert if a specific position (i,j) has the correct True/False value
- How to verify an entire row matches expected masking pattern

Current approach:
- Can print mask as string for visual inspection

Would appreciate examples of how to properly test these mask values.

**Comment by drisspg** [comment]
https://github.com/pytorch-labs/attention-gym/blob/36f8bd5ded5b3469f7892099590bb2405cc8f744/attn_gym/utils.py#L24

This function makes use of some private functions, and we dont provide BC guarantees on this but this should allow you to extract out the (potentially large) attn-scores matrix

**Comment by Leo-T-Zang** [comment]
Thanks @drisspg!

Will try!

**Comment by drisspg** [comment]
u
